twisted/test/test_memcache.py

```diff
@@ -278,11 +281,11 @@ def test_invalidGetResponse(self):
         If the value returned doesn't match the expected key of the current
         C{get} command, an error is raised in L{MemCacheProtocol.dataReceived}.
         """
-        self.proto.get("foo")
+        self.proto.get(b"foo")
         s = "spamegg"
         self.assertRaises(
             RuntimeError, self.proto.dataReceived,
-            "VALUE bar 0 %s\r\n%s\r\nEND\r\n" % (len(s), s))
+            networkString("VALUE bar 0 %d\r\n%s\r\nEND\r\n" % (len(s), s)))
```

Comment:
Actually, other tests hardcode the length. I'd be fine with that too, then it's consistent.